### PR TITLE
develop login-page navbar, enhanced the avartar and addressed issue #10

### LINKED ISSUE #11: develop navbar for login page
for your attention.
login

### PR DESCRIPTION
**Pull request by Wence17**
close #11 #12

### CODE REVIEW

**Comment by KMF21** [comment]
No screenshots? please provide screenshots.
Also, your PR should have a distinct title name and then your descriptions should come up in the body and not the title.
@Wence17 
well-done. cheers!

File: app/error/page.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+// import NavbarRow from "./components/Navbar";
+import Error404 from '../../components/Error404'
+import NavbarRow from '../../components/NavbarError'
+
+function ErrorPage() {
+  // baground = "bg-[#5ABF73] flex items-center justify-center "
+  return (
+    <div>
+      <NavbarRow />
```
Comment (KMF21) [inline app/error/page.tsx:10]: why is this still being called NavbarRow?

File: components/NavbarLogin.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <nav className="w-full bg-[#5ABF73] shadow-md">
+            <div className="flex items-center justify-between px-4 mx-auto lg:max-w-7xl md:items-center md:flex">
+
+                <div className="flex-1 flex justify-center mr-auto py-1.5 md:py-3 md:justify-start">
+                    <Link href="#">
+                        <h1 className=" text-white font-bold text-sm md:text-xl lg:text-2xl">Veritas Payment Portal</h1>
+                    </Link>
+                </div>
+            
+                <div className='flex mx-12 items-center justify-center pb-0 mt-0'>
```
Comment (KMF21) [inline components/NavbarLogin.tsx:17]: why flex and not flex-1?

**Review by KMF21 — CHANGES_REQUESTED** [review]

File: components/NavbarLogin.tsx (hunk 2 above)
Comment (Wence17) [reply components/NavbarLogin.tsx:17]: @KMF21 That's the middle navbar component. If I should flex-1, then it will be aligned to the left more.

**Pull request closed without merging**